<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decryptor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="CaesarCipherTab" name="caesarCipherTab">
       <attribute name="title">
        <string>Caesarova šifra</string>
       </attribute>
      </widget>
      <widget class="AffineCipherTab" name="affineCipherTab">
       <attribute name="title">
        <string>Afinní šifra</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="vigenereCipher">
       <attribute name="title">
        <string>Vigenèrova šifra</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Dešifrovaný text:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Šifrovaný text:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="encryptTextEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPlainTextEdit" name="decrpytTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionUkon_it"/>
   </widget>
   <widget class="QMenu" name="menuN_stroje">
    <property name="title">
     <string>Nástroje</string>
    </property>
    <addaction name="actionVypo_tat_index_koincidence"/>
   </widget>
   <widget class="QMenu" name="menuN_pov_da">
    <property name="title">
     <string>Nápověda</string>
    </property>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menuN_stroje"/>
   <addaction name="menuN_pov_da"/>
  </widget>
  <action name="actionVypo_tat_index_koincidence">
   <property name="text">
    <string>Vypočítat index koincidence</string>
   </property>
  </action>
  <action name="actionUkon_it">
   <property name="text">
    <string>Ukončit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CaesarCipherTab</class>
   <extends>QWidget</extends>
   <header>caesarciphertab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AffineCipherTab</class>
   <extends>QWidget</extends>
   <header>affineciphertab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
